index.html

```diff
@@ -1,29 +1,118 @@
 <!DOCTYPE html>
-<html lang="en-ca">
-  <head>
-    <!-- Set the page title -->
-    <title>Assignment 02</title>
+<html>
+<head>
 
-    <!-- Set the charset we're using to UTF-8 -->
-    <meta charset="UTF-8" />
+      <!-- Set the page title -->
+      <title>Website Landing Page</title>
 
-    <!-- Set the screens default zoom / scale to 1 -->
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    
-    <!-- Specifies the document mode for Internet Explorer -->
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      <!-- Set the charset we're using to UTF-8 -->
+      <meta charset="UTF-8" />
+  
+      <!-- Set the screens default zoom / scale to 1 -->
+      <meta name="viewport" content="width=device-width, initial-scale=1" />
+      
+      <!-- Specifies the document mode for Internet Explorer -->
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  
+      <!-- We are using a CSS Reset file to standardize browser default css -->
+      <link rel="stylesheet" href="reset.css" />
+  
+      <!-- This is where all of our awesome CSS code goes -->
+      <link rel="stylesheet" href="style.css" />
 
-    <!-- We are using a CSS Reset file to standardize browser default css -->
-    <link rel="stylesheet" href="reset.css" />
 
-    <!-- This is where all of our awesome CSS code goes -->
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <!--
 
-      YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
+ 
+</head>
+<body>
+  <div id="navbar">
+    <a href="home"><img src="https://pngimg.com/uploads/ufo/ufo_PNG71660.png" alt="My Logo" style="height: 22px;"></a>
+    <a href="#about">About</a>
+    <a href="#pre-order">Pre-Order</a>
+    <a href="#contact">Contact</a>
+  </div>
+  <div id='large-image'>
+    <img src="https://pngimg.com/uploads/ufo/ufo_PNG71660.png" alt="Large Image" width='25%' height='25%'>
+  </div>
+  <h1><span style="font-size: 65px">The </span><span style="font-size: 100px">U</span>niversal <span style="font-size: 100px">F</span>ighting <span style="font-size: 100px">O</span>rganization</h1>
+  <h2>The 1v1 fighting game for !@#$%^&*</h2>\
 
-    -->
-  </body>
+  <div class="container">
+    <img src="https://www.freepnglogos.com/uploads/alien-png/alien-fallout-the-vault-fallout-wiki-everything-25.png" alt="First Image" style="height: 500px;">
+    <button class="preorder-button">Preorder Now</button>
+    <img src="https://www.freepnglogos.com/uploads/alien-png/alien-fallout-the-vault-fallout-wiki-everything-25.png" alt="Second Image" class="flip-horizontal" style="height: 500px;">
+  </div>
```

Comment:
Although we haven't covered #a11y, try to use more descriptive `alt` text. 

"First / Second image" doesn't tell the user very much about what is inside those images.